string predicted_item_label
string predicted_waste_type
string actual_label
string pointcloud_topic
